.github/workflows/workflow.yml

```diff
@@ -8,19 +8,16 @@ on:
   pull_request:
```

Comment:
Should we change this to simply:
```
on: pull_request
```

This should essentially run the actions when a new PR is created, recreated or the Head branch of a PR is updated. Direct push to develop and master is not permitted anyway.